packages/account_sdk/src/signers/mod.rs

```diff
@@ -94,13 +94,12 @@ pub trait NewOwnerSigner: HashSigner {
         &self,
         chain_id: &Felt,
         contract_address: &Felt,
-        old_owner_guid: &Felt,
     ) -> Result<SignerSignature, SignError> {
         let message_hash = PedersenHasher::new(Felt::ZERO)
             .update(&selector!("change_owner"))
             .update(chain_id)
             .update(contract_address)
-            .update(old_owner_guid)
+            .update(contract_address)
```

Comment:
It used to be that when you want to `change_owner` you have to sign an `old_owner_guid` with your new signer in order to prove that you have control over the signer and dont loose control over the account. 
In the current context there is not really such a thing like an old_owner, since you could be adding one without removing the other.
As I understand it's not really important what you sign as long as it somehow proves that you have control over the new signer and won't loose the account. The contract address seemed like a natural choice.

Reply:
Ok got it, we are hashing it twice now which is why i was curious. We could probably remove it entirely but if that doesn't simplify or requires changing the argent components then let's keep it as is

Reply:
I've not noticed the double hashing. Fixed